x-pack/plugins/watcher/common/constants/action_states.js

```diff
@@ -33,4 +33,8 @@ export const ACTION_STATES = {
     defaultMessage: 'Error'
   }),
 
+  // Action has a configuration error
+  CONFIG_ERROR: i18n.translate('xpack.watcher.constants.actionStates.configErrorStateText', {
+    defaultMessage: 'Config Error'
```

Comment:
We try to use sentence case in Kibana. `Config error`.

Writing guidelines. https://elastic.github.io/eui/#/guidelines/writing